mitodl/open-discussions
# Refactor course_catalog course parsing

#### What are the relevant tickets?
closes #1838 
closes #1849

#### What's this PR do?
Refactor course sync to utilize custom serializers, to remove pylint exceptions, and to remove duplicated code.

#### How should this be manually tested?
Open the shell and manually run the various course sync functions (edx/mitx, ocw, bootcamps). Make sure that all of the data was correctly populated.
